src/lib/discord/commands/q.ts

```diff
@@ -0,0 +1,133 @@
+import { EmbedBuilder, ButtonStyle } from 'discord.js'
+import {
+  SlashCommandBuilder,
+  ActionRowBuilder,
+  ButtonBuilder,
+} from '@discordjs/builders'
+import { prisma } from '$lib/db'
+import type {
+  ButtonInteraction,
+  ChatInputCommandInteraction,
+  InteractionReplyOptions,
+  TextBasedChannel,
+} from 'discord.js'
+import type { Question } from '@prisma/client'
+
+function epoch(date: Date): number {
+  return Math.floor(date.getTime() / 1000)
+}
+
+function createDiscordTimestamp(date: Date): string {
+  return `<t:${epoch(date)}:R>`
+}
+
+function createQuestionEmbedFields(questions: Question[]) {
+  return questions.map((question) => ({
+    name: `${question.isSolved ? '✅' : '﹖'} ${createDiscordTimestamp(
+      question.createdAt
+    )}`,
+    value: `${question.title}\n[View in ${question.channelName}](${question.url})`,
+  }))
+}
+
+export const config = new SlashCommandBuilder()
+  .setName('q')
+  .setDescription('Get a digest of the latest questions')
```

Comment:
Since this only shows unsolved questions, should we call that out?

```suggestion
  .setDescription('Get a digest of the latest unsolved questions')
```

Reply:
oh good point, it was originally returning everything and I didn't come back to update the description